packages/cardano-services/src/Program/programs/providerServer.ts

```diff
@@ -137,17 +145,13 @@ const serviceMapFactory = (options: ServiceMapFactoryOptions) => {
     return new TypeormStakePoolProvider(args, { cache: getDbCache(), connectionConfig$, entities, logger });
   });
 
-  let networkInfoProvider: DbSyncNetworkInfoProvider | undefined;
-
   const getNetworkInfoProvider = (cardanoNode: CardanoNode, dbPools: DbPools) => {
```

Comment:
Is caching the `networkInfoProvider` not necessary anymore? There seemed to be a case where it was reused if already created.

Reply:
I had the same feeling as well, but I double checked: `getNetworkInfoProvider` is called only once in the entire SDK source.